Library/Formula/ruby.rb

```diff
@@ -56,7 +58,7 @@ def install
       args << "--with-arch=#{Hardware::CPU.universal_archs.join(",")}"
     end
 
-    args << "--program-suffix=23" if build.with? "suffix"
+    args << "--program-suffix=#{program_suffix}" if build.with? "suffix"
```

Comment:
Strictly speaking, it shouldn't be. (Haven't tested though, if `./configure` gets upset about a given but empty suffix.) But I think it looks nicer to skip this argument entirely, if it is not needed, instead of passing `--program-suffix=` for the majority of users. Deal-breaker?